src/services/GeoLocationService.ts

```diff
@@ -0,0 +1,33 @@
+import store from '@/store/store';
+import openWeather from '@/services/openWeatherMap';
+import { SET_CITY, SET_LAT, SET_LON } from '../store/mutations';
+
+export default class GeoLocationService {
+  static errorMessge: string = 'Unable to retrieve your location';
+
+  private output: object = {};
+
+  static getCurrentPosition(): Promise<any> {
+    // Possible TODO Create object for coords instead of using 'any'
```

Comment:
I ended up setting the return Promise type to 'any' so I wouldn't get errors trying to get the lat/long off of the object. I was getting "Property does not exist on Object" when using Promise<object>. I'm thinking there is a way to define the object specifying the properties but haven't looked into it

Reply:
I wondered about this. @bburggraff do you know if there are existing type definitions for standard browser APIs? Could we just import the types for `navigator.geolocation` somehow?